pdr_backend/pdr_dashboard/dash_components/plots.py

```diff
@@ -9,6 +11,26 @@
 from pdr_backend.util.time_types import UnixTimeS
 
 
+class FeedModalFigures:
+    def __init__(self):
+        self.sales_fig: go.Figure = create_figure(
+            [], "Sales", "Daily Sales (OCEAN)", False, ticker_format="%m-%d"
```

Comment:
we could also use an args config with this, but it's probably not worth it right now